### DISCUSSION TITLE
How to get global tilt irradiance by using diffuse and global irradiance?

### DISCUSSION THREAD

**Discussion by tjrk** [pvlib/pvlib-python#1817] [Q&A] [answered]
Hi there,

for a simulation, I need to derive the global tilt irradiance by using a dataset that consists of diffuse and global horizontal irradiance, measured at the specific location for every 15-minute step. 

How can I get the global tilt irradiation by using the tilt angle and azimut of the PV panel, as well as the specific location?

**Comment by jules-ch** [comment, marked as answer]
You'll need to use https://pvlib-python.readthedocs.io/en/stable/reference/generated/pvlib.irradiance.dni.html#pvlib.irradiance.dni to get the DNI from GHI and DHI.
You'll need to get `zenith` arrays from the `solar_position` module. 

Then you can the transposition module https://pvlib-python.readthedocs.io/en/stable/reference/generated/pvlib.irradiance.get_total_irradiance.html#pvlib.irradiance.get_total_irradiance to get the tilted irradiance providing the angles of your module
